layouts/Layout.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <ParentLayout>
+    <template #sidebar-bottom>
+      <section v-if="$themeConfig && $themeConfig.latestVersionPromo"
+        class="layout-note"
+      >
+        <p>
+          <a href="https://v3.docs.apostrophecms.org/">Learn about Apostrophe 3. 🆕</a>
+        </p>
+      </section>
+    </template>
+  </ParentLayout>
+</template>
+
+<script>
+import ParentLayout from '@parent-theme/layouts/Layout.vue'
+
+export default {
+  name: 'Layout',
+  components: {
+    ParentLayout
+  }
+}
+</script>
+
+<style lang="stylus" scoped>
+  .layout-note
+    background-color: $badgeTipColor
+    padding 0.35rem 1rem 0.35rem 1.25rem
+
+    a
+      color: white
```

Comment:
a bit ugly, can we try
```
background-color: #e3dbff;
display: inline-block;
border-radius: 10px;
padding: 0.35rem 1rem .35rem 0.5rem;
margin-left: 0.75rem;
font-size: 17px;
```

Wrap the 🌟 emoji in a span and place it to the left of the label, give it a margin-right of 0.5rem

result should look like
<img width="537" alt="image" src="https://user-images.githubusercontent.com/1889830/144062329-280a58cd-acf6-4e88-a30f-4a38b6ccf543.png">

Reply:
love it